src/components/GenericTopSection/GenericTopSection.jsx

```diff
@@ -1,7 +1,9 @@
+import { useRouter } from 'next/dist/client/router';
```

Comment:
question: can't you import from `next/router`?

Reply:
good catch 😄